docs-v2/integrations/all/bigcommerce.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: BigCommerce
+sidebarTitle: BigCommerce
+---
+
+API configuration: [`bigcommerce`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [How to register an Application](https://developers.tremendous.com/docs/oauth-20#step-1-register-a-developer-app)
+-   [OAuth-related docs](https://developer.bigcommerce.com/docs/integrations/apps/guide/auth)
+-   [List of OAuth scopes](https://developer.bigcommerce.com/docs/start/authentication/api-accounts#oauth-scopes)
+-   [API rate limiting](https://developer.bigcommerce.com/docs/start/best-practices/api-rate-limits)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+
+- BigCommerce offers various apps with different levels of visibility. For more details, see [BigCommerce types of Apps](https://developer.bigcommerce.com/docs/integrations/apps/guide/types)
+- After creating a new connection on Nango, you will need to rewrite your HTTP request headers to use the `X-Auth-Token` header to pass the API account's `access_token` instead of the Authorization header as described in [here](https://developer.bigcommerce.com/docs/start/authentication/api-accounts#how-to-migrate). Please look at the following example:
+
+```js
+        const connection = await nango.getConnection();
```

Comment:
what's up with the large indentation?

Reply:
I have updated this with the latest [commit](https://github.com/NangoHQ/nango/pull/2621/commits/8a639c929f2f0d94d57d40da3687feb50657f26c). Thank you for noting that 🙏.